packages/netlify-cms-widget-markdown/src/MarkdownControl/VisualEditor.js

```diff
@@ -141,15 +141,23 @@ export default class Editor extends React.Component {
   };
 
   handlePluginAdd = pluginId => {
+    const { getEditorComponents } = this.props;
     const { value } = this.state;
     const nodes = [Text.create('')];
+    const pluginDefinition = getEditorComponents().get(pluginId);
+
+    const defaultValues = {};
+    pluginDefinition.get('fields', List()).forEach(field => {
+      let defaultValue = field.get('default');
+      defaultValues[field.get('name')] = defaultValue;
+    });
     const block = {
       kind: 'block',
       type: 'shortcode',
       data: {
         shortcode: pluginId,
         shortcodeNew: true,
-        shortcodeData: Map(),
+        shortcodeData: new Map(defaultValues),
```

Comment:
We shouldn't use `new` here. I'd say let's just create a deep immutable structure in the `forEach` above, in case there are object widgets with default values, and just pass that in.

Reply:
@erquhart Would you mind just pushing that?

Reply:
@floydnoel Thanks, but @erquhart has it done already. If you'd like to review the PR, that would be great!